ANTLR-version/App.g4

```diff
@@ -1,11 +1,23 @@
 // Define grammar
 grammar App;
-s : comment   | EOF   ;
-comment : COMMENT NEWLINE s;
+s : comment | definition | NEWLINE s | EOF  ;
+
+comment : COMMENT s;
+definition : ID ASSIGN_OPERATORS equation s ;
+conditional_statement: equation CONDITION equation;
+
+value : ID | NUMBER | '"' ~('"')* '"' ;
```

Comment:
This should also include booleans ("True" | "False")

Reply:
A conditional statement can technically be set to a value.

Reply:
> A conditional statement can technically be set to a value.

equation supports assigning to a single value.

Reply:
> This should also include booleans ("True" | "False")

This is not required in the sample python file